CHANGELOG.md

```diff
@@ -21,6 +21,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - -Werror: Turn warnings into errors.
   - -Werror=<option>: Turn the specified warning into an error.
 - oesign sign now allows option -o/--output-file, to specify location to write signature of enclave image.
+- Debugger Contract has been extended to support multiple modules.
+  - Refer to [design document](docs/DesignDocs/DebuggerSupportForMultiModuleEnclaves.md) for details.
+  - `path` fields in `oe_debug_enclave_t` and `oe_debug_module_t` are now defined to be in
```

Comment:
Consider moving this under the `### Changed` header. While the feature is largely additional, the specific modification here that requires developer attention might be better surfaced there.